src/components/buttons/CollectionSelector.vue

```diff
@@ -46,7 +60,10 @@ export default {
   data: () => {
     return {
       collections: [],
-      identifier: ''
+      identifier: '',
+      faStyle: {
```

Comment:
better pass this as a property, so it can be changed outside. You can use this as a 'default'